### PR TITLE
access explore view completed

### PR DESCRIPTION
**Pull request by Tushar200018**

### CODE REVIEW

File: backend/rules/processgraph/build_graph.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ func ProcessGraphPathResult(records neo4j.Result, pathKeyStr string) (types.Grap
 
 	for records.Next() {
 		record := records.Record()
+
```
Comment (vishjain) [inline backend/rules/processgraph/build_graph.go:35]: Remove this blank space.

File: backend/handlers/get_edge_info.go [hunk 2]
```diff
@@ -34,55 +70,102 @@ func getRelationshipReason(tx neo4j.Transaction, edgeID int) (neo4j.Result, erro
...
-func getPrivelegeEscalationReason(tx neo4j.Transaction, edgeID int) (interface{}, error) {
+func castLabelLst(labels interface{}) ([]string, error) {
+
+	var resutLabelLst []string
+	labelLst, ok := labels.([]interface{})
+	if !ok {
+		return nil, fmt.Errorf("labels must be a slice of strings")
+	}
+	for _, label := range labelLst {
+		labelStr, _ := label.(string)
```
Comment (varunjain99) [inline backend/handlers/get_edge_info.go:155]: check this is valid and return error if not?

File: backend/rules/processgraph/build_graph.go (hunk 1 above)
Comment (Tushar200018) [reply backend/rules/processgraph/build_graph.go]: done

File: backend/handlers/get_edge_info.go (hunk 2 above)
Comment (Tushar200018) [reply backend/handlers/get_edge_info.go]: done

**Pull request merged**